source/extensions/filters/http/jwt_authn/jwt_cache.cc

```diff
@@ -14,6 +14,8 @@ namespace {
 
 // The default number of entries in JWT cache is 100.
 constexpr int kJwtCacheDefaultSize = 100;
+// The maximum size of JWT to be cached.
+constexpr int MAX_JWT_SIZE = 4 * 1024; // 4KiB
```

Comment:
Google c++ coding style prefer this variable name

"kMaxJwtSizeForCache"

Reply:
Changed style.